esda/shape.py

```diff
@@ -463,43 +471,110 @@ def nmi(collection):
 
 def second_areal_moment(collection):
     """
-    Using equation listed on en.wikipedia.org/Second_Moment_of_area, the second
-    moment of area is actually the cross-moment of area between the X and Y
-    dimensions:
+    Using equation listed on en.wikipedia.org/wiki/Second_moment_of_area#Any_polygon, the second
+    moment of area is the sum of the inertia across the x and y axes:
 
+    The :math:`x` axis is given by:
     .. math::
-        I_xy = (1/24)\\sum^{i=N}^{i=1} (x_iy_{i+1} + 2*x_iy_i + 2*x_{i+1}y_{i+1} +
-        x_{i+1}y_i)(x_iy_i - x_{i+1}y_i)
+        I_x = (1/12)\\sum^{N}_{i=1} (x_i y_{i+1} - x_{i+1}y_i) (x_i^2 + x_ix_{i+1} + x_{i+1}^2)
 
-    where x_i, y_i is the current point and x_{i+1}, y_{i+1} is the next point,
-    and where x_{n+1} = x_1, y_{n+1} = 1.
+    While the :math:`y` axis is in a similar form:
+        I_y = (1/12)\\sum^{N}_{i=1} (x_i y_{i+1} - x_{i+1}y_i) (y_i^2 + y_iy_{i+1} + y_{i+1}^2)
```

Comment:
I suppose we may want this within `.. math::` block?

Reply:
Good catch.